WISE-Community/WISE
# Let teachers set and edit run end dates

To test:
- Log in as a teacher
- Edit the start and end dates for some runs in your Class Schedule (via "Edit Settings" in the run menu)
- Make sure the dates update correctly and the run details update on the page
- Set up some new runs both with and without end dates, make sure they are created successfully
- Make sure you can't set the end date before the start date or start date after the end date
- Check to see that the last student login date is displayed for runs that have been used by students
- Log in as a student and make sure all students runs display correctly and can be launched

Closes #1789.
